uber/jinja.py

```diff
@@ -0,0 +1,61 @@
+from uber.common import *
+
+
+class JinjaEnv:
+    _env = None
+    _exportable_functions = {}
+    _filter_functions = {}
+    _template_dirs = []
+
+    @staticmethod
+    def append_template_dir(dirname):
+        """
+        Add another template directory we should search when looking up templates.
+        """
+        JinjaEnv._template_dirs.insert(0, dirname)
+
+    @staticmethod
+    def env():
```

Comment:
hey actually related question then:  do we have any good place to stick initialization code that runs on startup?  there's a few other places in the codebase where we could benefit from having an explicit initialize() called on startup.  for instance, twilio initializes at import time, vs probably should initialize later on in the sequence once everything's been imported and settings have been messed with.

i.e. would be good if we:
- imported and self-registered everything from plugins, in whatever order that happens
- call some initialize() function which gets hooked in